index.js

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+*/
+'use strict';
+
+const program = require('commander');
+const {api, key, testApi} = require('./lib');
+
+// debuggers
+// const loaderDebug = require('debug')('vc:loader');
+
+// function inc(value, total) {
+//   return total + 1;
+// }
+
+/*
+if(process.env.JSONLD_LOADER_MAP) {
+  const map = {};
+  const cache = {};
+  const oldLoader = vc.documentLoader;
+  process.env.JSONLD_LOADER_MAP
+    .split(',')
+    .map(v => v.split('='))
+    .forEach(v => map[v[0]] = v[1]);
+  loaderDebug('map: %O', map);
+  vc.documentLoader = async (url) => {
+    if(url in cache) {
+      // already loaded
+      loaderDebug('from cache: %s', url);
+    } else if(url in map) {
+      // load from file
+      loaderDebug('from file: %s', url);
+      cache[url] = {
+        contextUrl: null,
+        documentUrl: url,
+        document: JSON.parse(await fs.readFile(map[url]))
+      };
+    } else {
+      // load with default loader
+      loaderDebug('default: %s', url);
+      cache[url] = await oldLoader(url);
+    }
+    return cache[url];
+  };
+}
+*/
+
+// program
+//   .version(require('../package.json').version)
+//   .option('-v, --verbose', 'Increase verbosity', inc, 0);
+
+program
+  .command('generate-key')
+  .description('Generate an Ed25519 key pair')
+  .option('-t, --token <token>', 'GitHub personal access token.')
+  .option('-s, --secp256k1', 'Generate a Secp256KeyPair.')
+  .action(key.generate);
+
+program
+  .command('issue')
+  .alias('i')
+  .description('issue a credential')
+  .option('-k, --key <key-file>')
+  // .option('-i, --issuer <issuer>', 'URL for the issuer public key.')
+  // .option('-r, --private-key <private-key>', 'Private key PEM file.')
+  // .option('-d, --date <date>', 'Signature date [now].', new Date())
+  // .option('-D, --domain <domain>', 'Signature domain [none].')
+  // .option('-a, --algorithm <algorithm>',
+  //   'Signature algorithm [Ed25519Signature2018].')
+  .action(api.issue)
+  .on('--help', () => {
+    console.log();
+    console.log('  Examples: ');
+    console.log();
+    console.log('    $ vc-js issue ... < cred.jsonld');
+    console.log();
+  });
+
+program
```

Comment:
Nuke test-issue from orbit. It never should have been in vc-js, and this just compounds the problem.

@dmitrizagidulin, this is exactly why we don't let test suite stuff leak into libraries/binaries. Inevitably, well meaning developers (like @mattcollier) come along and assume that it's a normal part of the binary, and then the problem is now in two places instead of one. Any sort of test-issue or test-presentation stuff should have been in the test suite and only the test suite.

Reply:
Not sure I understand.. Was `test-issue` used here before? (Not seeing it at the moment).

Reply:
@dmitrizagidulin it's was removed.